src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py

```diff
@@ -193,17 +193,17 @@ def load_arguments(self, _):
 
     with self.argument_context('vm disk') as c:
         c.argument('vm_name', options_list=['--vm-name'], id_part=None, completer=get_resource_name_completion_list('Microsoft.Compute/virtualMachines'))
-        c.argument('disk', validator=validate_vm_disk, help='disk name or ID', completer=get_resource_name_completion_list('Microsoft.Compute/disks'))
+        c.argument('disk', validator=validate_vm_disk, help='The disk name or ID', completer=get_resource_name_completion_list('Microsoft.Compute/disks'))
```

Comment:
Within the `az disk` subgroup, disk should be aliased to `--name`, not `--disk`.

Reply:
Hello @tjprescott, I did not make any changes to `az disk` in this PR. I did make changes to `az vm disk` and `az vm unmanaged-disk`.

Although I would have preferred `--name -n`, I changed the (disk name) options from `--name -n` to `--disk` for both command groups because of [az vmss disk attach](https://docs.microsoft.com/en-us/cli/azure/vmss/disk?view=azure-cli-latest#az-vmss-disk-attach), which uses **both** `--disk` and `--name -n`. This means I cannot deprecate `--disk` to `--name -n` for this command. So I had to choose between consistent naming for `az vm disk`, `az vmss disk`, `az vm unmanaged-disk` or using `--name` for the vm disk commands.

Reply:
In all three cases, the alias should be `--name`. If there is something that uses `--disk` it should be deprecated.